src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -69,6 +69,40 @@ case class Entity(
   def path : String = workspaceName.path + "/entities/" + name
 }
 
+@ApiModel(value = "Method configuration name")
+case class MethodConfigurationName(
+                   @(ApiModelProperty@field)(required = true, value = "The name of the method configuration")
+                   name: String,
+                   @(ApiModelProperty@field)(required = true, value = "This method configuration's owning workspace")
+                   workspaceName: WorkspaceName
+                   )
+
+@ApiModel(value = "Method")
+case class Method(
+                   @(ApiModelProperty@field)(required = true, value = "The name of the method")
+                   name: String,
+                   @(ApiModelProperty@field)(required = true, value = "The namespace of the method")
+                   nameSpace: String,
+                   @(ApiModelProperty@field)(required = true, value = "The version of the method")
+                   version: String
+                 )
+@ApiModel(value = "Method Configuration")
+case class MethodConfiguration(
```

Comment:
the method configs themselves should have a namespace in addition to that of the workspace, I meant you should add that namespace.